test/linter/test_solhint.vader

```diff
@@ -2,7 +2,7 @@ Before:
   call ale#assert#SetUpLinterTest('solidity', 'solhint')
   runtime autoload/ale/handlers/solhint.vim
 
-  let b:args = ' --formatter compact %s'
```

Comment:
Modify the test to check command is correct depending on the version of solhint used. You can see the `test_ansible_lint.vader` test for example on how to do this.